### DISCUSSION TITLE
How to hack an Android game?

### DISCUSSION THREAD

**Discussion by btimofeev** [btimofeev/UniPatcher#44] [Q&A] [answered]
I am often asked how to hack an Android game using UniPatcher.

**Comment by btimofeev** [comment, marked as answer]
UniPatcher was never intended to hack Android games. You should look for some other app.

**Reply by Extomi** [reply]
Ok show mea other app

**Reply by btimofeev** [reply]
I have not used such software. Google on this topic =)
